IFC4x3/Sections/Domain specific data schemas/Schemas/IfcRailDomain/PropertySets/Pset_CableSegmentTypeWirePairSegment/DocPropertySet.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertySet xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="Pset_CableSegmentTypeWirePairSegment" UniqueId="836e0a75-80b5-d43b-0ccd-892ddace6c3b" ApplicableType="IfcCableSegment/WIREPAIRSEGMENT" PropertySetType="PSET_TYPEDRIVENOVERRIDE">
+	<Properties>
+		<DocProperty xsi:nil="true" href="CharacteristicImpedance_10mgUNnHf1VfAg7w_L3oMi" />
+		<DocProperty xsi:nil="true" href="ConductorDiameter_2LgQvvudb6UBQkAzoEIL3k" />
+		<DocProperty xsi:nil="true" href="CoreConductorDiameter_3ZwqVh2H99qOG4Rj3QX" />
+		<DocProperty xsi:nil="true" href="JacketColor_1v79lnGQHF78BAfCNIBvyf" />
+		<DocProperty xsi:nil="true" href="SequentialNumber_1rZx8VDW987QxAxZ64yNea" />
```

Comment:
Seems like an instance property

Reply:
Changed sequential number to Pset_cableSegmentOccurrence. commit : 4d5e83c1bf208182fa5e5d0b2648e61d35e95f5a